#True if left pole of gate is seen
bool detectedLeftPole
#Angle from vertical centre line of hsv_filter to detected left pole, 0 if not seen
float32 angleLeftPole
#Distance from robot to left pole, 0 if not seen
float32 distanceLeftPole
#True if right pole of gate is seen
bool detectedRightPole
#Angle from vertical centre line of hsv_filter to detected right pole, 0 if not seen
float32 angleRightPole
#Distance from robot to right pole, 0 if not seen
float32 distanceRightPole
#True if top pole of gate is seen
bool detectedTopPole
#Angle from horizontal centre line of hsv_filter to detected top pole, 0 if not seen
float32 angleTopPole
#Distance from robot to top pole, 0 if not seen
float32 distanceTopPole